Manifest.toml

```diff
@@ -949,6 +967,12 @@ git-tree-sha1 = "838a3a4188e2ded87a4f9f184b4b0d78a1e91cb7"
 uuid = "ae029012-a4dd-5104-9daa-d747884805df"
 version = "1.3.0"
 
+[[deps.Revise]]
```

Comment:
It's better to add Revise to your default Julia environment and [use it by default](https://timholy.github.io/Revise.jl/v0.6/config.html#Using-Revise-by-default-1). Once you set that up, you can remove it from the dependencies here.